client/src/VolunteerDashboard_Comps/ViewClass.js

```diff
@@ -1,14 +1,14 @@
 import React from "react";
 import { useNavigate } from "react-router-dom";
-import "./ViewClass.css";
+import "C:/Users/Dewayne/Documents/CYF/cyf-student-register/client/src/index.css"
```

Comment:
I dont think that this is the correct file path. Noone else can access it it should have a file path relating to the repo